camelAssemblies/README.md

```diff
@@ -0,0 +1,155 @@
+# Overview
+
+This document outlines the contents of the `CamelAssemblies` subproject within the extension sources repo.
+
+The `CamelAssemblies` subproject creates Vantiq projects from
+[Camel Kamelet Definitions](https://camel.apache.org/camel-kamelets/3.21.x/). These projects will
+be made available in a Vantiq Public Catalog as _Vantiq assemblies_. These assemblies, when imported,
+create Vantiq services and sources for connections to or from external systems supported by Apache Camel.
+
+# Prerequisites <a name="pre" id="pre"></a>
+
+**IMPORTANT:** A basic understanding of the [Vantiq Camel Connector](../camelConnector/README.md) is assumed.
+
+All assemblies contained herein (Vantiq Camel assemblies) are built atop the base Camel Connector assembly, 
+`com.vantiq.extsrc.camelconn.camelConnector`.  Specifically, the Camel Connector 
+assembly defines the Camel Connector source implementation type and the schema type
+`com.vantiq.extsrc.camelcomp.message` that defines the message format for messages used between the Camel connector
+and the Vantiq system.
+
+# Repository Contents
+
+Apache Camel, briefly, is organized into a set of _components_ that are used to integrate with other software 
+systems. The Vantiq Camel Connector provides the ability to load the components necessary to operate _routes_, 
+Camel's means for describing the interaction between information systems.
+
+Apache Kamelets are packages containing the routes and associated settings needed to set up connections using Apache
+Camel-K. 
+
+The assemblies contained in this repo take the information contained in the Kamelet definitions, 
```

Comment:
there are no assemblies "contained in this repo" at first glance.   
could you add a clarification here?   Maybe:
_This subproject will generate Vantiq assemblies from the camel definitions located <whereever they are>.  To generate the assembles, just run `../gradlew assemble`.  After that, the Vantiq assemblies will be located in zip files under the `build/distributions` directory. For example, build/distributions/ftp_sink_v3_21_0.zip_.